opencog/nlp/relex2logic/rules/definite.scm

```diff
@@ -18,24 +19,23 @@
                 (VariableNode "$noun")
                 (VariableNode "$a-parse")
             )
-	    (InheritanceLink
-   		(VariableNode "$noun")
-   		(DefinedLinguisticConceptNode "definite")
-	    )
+            (InheritanceLink
+                (VariableNode "$noun")
+                (DefinedLinguisticConceptNode "definite")
+            )
         )
-       (ListLink
-        (ExecutionOutputLink
-       	   (GroundedSchemaNode "scm: pre-definite-rule")
-       	      (ListLink
-       	         (VariableNode "$noun")
+        (ListLink
+            (ExecutionOutputLink
+                (GroundedSchemaNode "scm: pre-definite-rule")
+                (ListLink (VariableNode "$noun"))
             )
         )
-      )
     )
 )
 
-; This is function is not needed. It is added so as not to break the existing
-; r2l pipeline.
+; This is function is not needed. It is added so as not to break the
+; existing r2l pipeline.  Huh ??? How can it not be needed?  It is
```

Comment:
The wording is misleading I guess. What is meant is `definite-rule` could be modified to accept a node instead of the name of the node. But doing that at the time of adding this rule was cumbersome, as it meant modifying the java code that generated the call in the previous r2l-implementation. In short it is a "temporary hack"